lib/solidus_subscriptions/configuration.rb

```diff
@@ -80,5 +80,9 @@ def subscribable_class
     def churn_buster?
       churn_buster_account_id.present? && churn_buster_api_key.present?
     end
+
+    def ignore_past_failed_installments_on_new_cycle?
+      @ignore_past_failed_installments_on_new_cycle ||= false
+    end
```

Comment:
I don't think you'll need this: `nil` is treated as `false`, so you can just check `ignore_past_failed_installments_on_new_cycle` directly.